app/src/main/res/values-es/strings.xml

```diff
@@ -336,84 +390,158 @@
   <string name="lnurl_withdraw_confirm_button">Retirar</string>
-  <string name="lnurl_withdraw_wait">Solicitando fondos…</string>
+  <string name="lnurl_withdraw_wait">Retirando fondos…</string>
   <string name="lnurl_withdraw_success">Por favor espera mientras %1$s está enviando fondos.\n\nMantén la aplicación en línea.</string>
-  <string name="lnurl_withdraw_error_internal">No se pudo procesar esta solicitud de retiro. Por favor inténtalo de nuevo.</string>
-  <string name="lnurl_withdraw_error_invalid">Esta solicitud de retiro no es válida. Por favor, inténtalo de nuevo.</string>
+  <string name="lnurl_withdraw_error_internal">No se pudo procesar la solicitud de retirada. Por favor inténtalo de nuevo.</string>
+  <string name="lnurl_withdraw_error_invalid">Esta solicitud de retirada no es válida. Por favor, inténtalo de nuevo.</string>
   <string name="lnurl_withdraw_error_amount">El importe no es válido.</string>
   <string name="lnurl_withdraw_error_amount_min">El importe debe ser de al menos %1$s.</string>
   <string name="lnurl_withdraw_error_amount_max">El importe no puede exceder %1$s.</string>
   <string name="lnurl_withdraw_error_remote">%1$s respondió con un error: %2$s.\n\nVuelve a intentarlo más tarde o contacta a su soporte.</string>
-  <string name="lnurl_withdraw_error_remote_invalid_response">Datos inválidos</string>
+  <string name="lnurl_withdraw_error_remote_invalid_response">Datos incorrectos</string>
 
   <!-- //////////////// display preferences //////////////// -->
 
   <string name="prefs_display_title">Opciones de visualización</string>
-  <string name="prefs_display_coin_label">Unidad Bitcoin</string>
+  <string name="prefs_display_coin_label">Unidad de Bitcoin</string>
   <string name="prefs_display_fiat_label">Moneda fiduciaria</string>
   <string name="prefs_display_theme_label">Tema de la aplicación</string>
 
   <!-- //////////////// electrum server //////////////// -->
 
   <string name="electrum_title">Servidor Electrum</string>
-  <string name="electrum_subtitle">Por defecto, Phoenix se conecta a servidores electrum aleatorios para acceder a la blockchain de Bitcoin. También puedes optar por conectarte a tu propio servidor Electrum.</string>
+  <string name="electrum_subtitle">Por defecto, Phoenix se conecta a servidores Electrum aleatorios para acceder a la cadena de bloques de Bitcoin. También puedes optar por conectarte a tu propio servidor Electrum.</string>
   <string name="electrum_change_button">Establecer servidor…</string>
-  <string name="electrum_block_height_label">Altura del bloque</string>
+  
+  <string name="electrum_block_height_label">Altura de bloque</string>
   <string name="electrum_tip_label">Marca de tiempo</string>
-  <string name="electrum_fee_rate_label">Tasa de tarifa</string>
-  <string name="electrum_xpub_label">Clave maestra pública</string>
+  <string name="electrum_fee_rate_label">Ratio de comisión</string>
```

Comment:
Please consider

`Tasa de comisión`
`Comisión proporcional`

or some other alternative, as ratio is not even a Spanish word.

Reply:
>It is! Widely used in science and understood also colloquially: https://dle.rae.es/ratio

O M G

Reply:
I have seen a lot of "ratio de conversión" over there, but I always thought it was an anglicism.
Even now, I still think it was recently included or something. xP

I also understand the confusion about "tasa" being used in some parts as "comisión", so let's go with whatever you think it's the best choice.

Reply:
Resolved previously!